scripts/install_third_party_libs.py

```diff
@@ -22,19 +22,39 @@
 import subprocess
 import sys
 
-# These libraries need to be installed before running or importing any script.
+
 TOOLS_DIR = os.path.join(os.pardir, 'oppia_tools')
-# Download and install pyyaml.
-if not os.path.exists(os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')):
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', 'pyyaml==5.1.2', '--target',
-        os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')])
-
-# Download and install future.
-if not os.path.exists(os.path.join('third_party', 'future-0.17.1')):
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', 'future==0.17.1', '--target',
-        os.path.join('third_party', 'future-0.17.1')])
+
+
+def install_prerequisites():
```

Comment:
Why is it necessary to wrap this in a function?

Reply:
I guess it makes the code easy to read as its divided into blocks based on what it does..

Reply:
In that case, move this function lower in the line (below line 79) and call it in `main`.

Reply:
You'll have to make sure that it doesn't break anything; otherwise, it's better to have it the way it was before.

Reply:
Since you can't call this function in `main`, there isn't a good reason to wrap this in a function. Please extract it to what it was before.

Reply:
Done! I removed it.